packages/decentraland-ecs/src/ecs/Engine.ts

```diff
@@ -383,14 +403,16 @@ export class Engine {
         this.checkRequirements(entity, componentGroups[i])
       }
     }
+
+    return this
   }
 
   private componentRemovedHandler(event: ComponentRemoved) {
     // In case a single component gets removed from an entity, we inform
     // all systems that this entity lost this specific component.
     const { entity, componentName } = event
 
-    if (!entity.isAddedToEngine()) return
+    if (!entity.isAddedToEngine()) return this
```

Comment:
Private and unnecesary
```suggestion
    if (!entity.isAddedToEngine()) return
```